client/src/components/Button/SubmitButton.js

```diff
@@ -7,48 +7,6 @@ const { button } = theme;
 
 const SubmitButton = styled(BaseButton).attrs(({ size, inline }) => {
```

Comment:
Hmm, looks like we don't need SubmitButton anymore. For now I guess we can leave this file as it but we should maybe remove this altogether in the future.